app/views/users/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if flash[:success] %>
+    <p><%= flash[:success] %></p>
+<% end %>
+
+<h1> <%= @user.name  %> </h1>
+
+<h2>Attended Events</h2>
```

Comment:
- [ ] Show also the events created by the user in their show page. 
- [ ] Prefer sending @upcoming_events and @past-events from the controller instead of accesing @user.attended_events.upcoming, etc directly in the view.